public/app/core/components/Page/types.ts

```diff
@@ -22,15 +24,11 @@ export interface PageProps extends HTMLAttributes<HTMLDivElement> {
   /** Control the page layout. */
   layout?: PageLayoutType;
   /**
+   * TODO: Not sure we should deprecated it given the sidecar project?
    * @deprecated this will be removed when bodyScrolling is enabled by default
    * Can be used to get the scroll container element to access scroll position
    * */
-  scrollRef?: RefCallback<HTMLDivElement>;
-  /**
-   * @deprecated this will be removed when bodyScrolling is enabled by default
-   * Can be used to update the current scroll position
-   * */
-  scrollTop?: number;
+  onSetScrollRef?: (ref: ScrollRefElement) => void;
```

Comment:
only DashboardPage ever uysed scrollTop and scrollRef (checked enterprise as well), so safe to just remove / rename